libcxx/include/__type_traits/reference_constructs_from_temporary.h

```diff
@@ -0,0 +1,42 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCPP___TYPE_TRAITS_REFERENCE_CONSTRUCTS_FROM_TEMPORARARY_H
+#define _LIBCPP___TYPE_TRAITS_REFERENCE_CONSTRUCTS_FROM_TEMPORARARY_H
+
+#include <__config>
+#include <__type_traits/integral_constant.h>
+
+#if !defined(_LIBCPP_HAS_NO_PRAGMA_SYSTEM_HEADER)
+#  pragma GCC system_header
+#endif
+
+_LIBCPP_BEGIN_NAMESPACE_STD
+
+#if !defined(_LIBCPP_CXX03_LANG)
+
+template <class _Tp, class _Up>
+inline constexpr bool __reference_constructs_from_temporary_v = __reference_constructs_from_temporary(_Tp, _Up);
```

Comment:
I think `const` can be used instead to make this usable in C++03 mode. `__reference_constructs_from_temporary_v<T, U>` will still be usable in constant expressions as it's of type `const bool`.

Reply:
I might have been confused here. Do you mean the frozen C++03 headers have nothing to do with the C++03 mode?

Reply:
I'm not sure. Perhaps the "C++03 with non-frozen headers" mode is still being tested.

Reply:
You are right. This makes sense. Thanks.